example/deploy-docs-live-code-contracts.ts

```diff
@@ -0,0 +1,969 @@
+import { contractMapBigMap } from './data/contractMapInitialStorage';
+import { contractJson } from './data/contractJson';
+import { tokenCode, tokenInit } from '../integration-tests/data/tokens';
+import { fa2ForTokenMetadataView } from '../integration-tests/data/fa2-for-token-metadata-view';
+import { tacoContractTzip16 } from '../integration-tests/data/modified-taco-contract';
+import {
+  contractCode,
+  metadataViewsExample1,
+  metadataViewsExample2,
+} from '../integration-tests/data/metadataViews';
+import { contractMap8pairs } from './data/contractMap8pairs';
+import { char2Bytes } from '@taquito/utils';
+
+const provider = 'https://hangzhounet.api.tez.ie';
+export const signer: any = new InMemorySigner(
+  'edskRtmEwZxRzwd1obV9pJzAoLoxXFWTSHbgqpDBRHx1Ktzo5yVuJ37e2R4nzjLnNbxFU4UiBU1iHzAy52pK5YBRpaFwLbByca'
+);
+export const tezos = new TezosToolkit(provider);
+
+let contract_catalogue = new Map();
+
+let user_addresses = new Map();
+user_addresses.set('TestFunder', 'tz1bwsEWCwSEXdRvnJxvegQZKeX5dj6oKEys');
+user_addresses.set('Alice', 'tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb');
+user_addresses.set('Deborah', 'tz1h3rQ8wBxFd8L9B3d7Jhaawu6Z568XU3xY');
+user_addresses.set('Eddy', 'tz3WXYtyDUNL91qfiCJtVUX746QpNv5i5ve5');
+user_addresses.set('Freda', 'tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx');
+user_addresses.set('Glen', 'tz1PgQt52JMirBUhhkq1eanX8hVd1Fsg71Lr');
+user_addresses.set('Validation', 'tz1L9r8mWmRPndRhuvMCWESLGSVeFzQ9NAWx');
+user_addresses.set('WalletReceiver', 'tz1NhNv9g7rtcjyNsH8Zqu79giY5aTqDDrzB');
+user_addresses.set('Allowances', 'tz1Nu949TjA4zzJ1iobz76fHPZbWUraRVrCE');
+user_addresses.set('BigMapLedger', 'tz1XTyqBn4xi9tkRDutpRyQwHxfF8ar4i4Wq');
+
+const min_balance = 100;
+
+async function checkBalances() {
+  console.log('checking funds...');
+  try {
+    user_addresses.forEach(async (value: string, key: string) => {
+      const user_balance: any = await tezos.tz.getBalance(value);
+      if (user_balance < min_balance) {
+        tezos.setSignerProvider(signer);
+        console.log(`${key} has a too-low balance of ${user_balance / 1000000}.`);
+        console.log(`Transfering ${min_balance} ꜩ to ${key}...`);
+        await transferMinBalance();
+        console.log(`Transfer complete`);
+      } else {
+        console.log(`${key} has an ok balance of ${user_balance / 1000000}.`);
+      }
+
+      async function transferMinBalance() {
+        await tezos.contract
+          .transfer({ to: value, amount: min_balance })
+          .then((op) => {
+            console.log(`Waiting for ${op.hash} of transfer to ${value} to be confirmed...`);
+            return op.confirmation(1).then(() => op.hash);
+          })
+          .catch((error) => console.log(`Error: ${error} ${JSON.stringify(error, null, 2)}`));
+      }
+    });
+  } catch (ex) {
+    console.error(ex);
+  }
+}
+
+async function originateTheContracts() {
+  checkBalances();
+
+  console.log('originating...');
+   tslint:disable-next-line: no-floating-promises
```

Comment:
`tslint:disable-next-line: no-floating-promises` is missing `//`
But I think this line can be removed.

Reply:
removed.